contribs/gmf/examples/objecteditinghub.js

```diff
@@ -1,17 +1,18 @@
 goog.provide('gmfapp.objecteditinghub');
 
 goog.require('goog.asserts');
-/** @suppress {extraRequire} */
-goog.require('ngeo.proj.EPSG21781');
+goog.require('gmf.editing.XSDAttributes');
 goog.require('gmf.ObjectEditingManager');
 goog.require('gmf.theme.Themes');
-goog.require('gmf.XSDAttributes');
+/** @suppress {extraRequire} */
+goog.require('ngeo.proj.EPSG21781');
```

Comment:
For curiosity, do you know why this projection is required? It is not used in the file.

Reply:
no idea... there is no map in this file :-/
Perhaps an artifact. I'll remove it in my next commit.